workspaces/adventure-pack/goodies/typescript/Iterator.from/index.test.ts

```diff
@@ -0,0 +1,91 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
```

Comment:
To make sure we're testing our own `Iterator.from` and not the built-in, for versions of Node that have it, we should add:

```ts
(globalThis as Record<string, unknown>).Iterator && delete ((globalThis as Record<string, unknown>).Iterator as Record<string, unknown>).from;
```

(similar to some other goodies)

Reply:
ack!